### ISSUE TITLE
Test failure with Matplotlib 3.8.0rc1

### ISSUE THREAD

**Issue by QuLogic** [boutproject/animatplot-ng#13]
I'm [trying out Matplotlib 3.8.0rc1](https://copr.fedorainfracloud.org/coprs/qulogic/mpb.1/build/6335387/) and there's 1 test failure [that doesn't occur with 3.7.2](https://copr.fedorainfracloud.org/coprs/qulogic/mpb.1.checker/build/6335495/):

```
_________________________ TestComparisons.test_Surface _________________________

args = (<tests.test_blocks.TestComparisons object at 0x7f6be24cbe30>,)
kwargs = {}, anim = <animatplot.animation.Animation object at 0x7f6be23a25d0>
fignum = 1, fig = <Figure size 640x480 with 1 Axes>

    @functools.wraps(func)
    def wrapper(*args, **kwargs):
        # First close anything from previous tests
        plt.close("all")
    
        anim = func(*args, **kwargs)
        if remove_text:
            fignum = plt.get_fignums()[0]
            fig = plt.figure(fignum)
            remove_ticks_and_titles(fig)
        try:
>           _compare_animation(anim, baseline_images, fmt, nframes, tol)

tests/tools.py:78: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

anim = <animatplot.animation.Animation object at 0x7f6be23a25d0>
expected = 'Blocks/Surface', format_ = '.png', nframes = 3, tol = 0.001

    def _compare_animation(anim, expected, format_, nframes, tol):
        # generate images from the animation
        base_dir, filename = split(join("tests", "baseline_images", expected))
        out_dir = split(join("tests", "output_images", expected))[0]
    
        if not os.path.exists(out_dir):
            os.makedirs(out_dir)
    
        anim.save(os.path.join(out_dir, (filename + format_)), writer=BunchOFiles())
    
        for i in range(nframes):
            image_name = "%s%d%s" % (filename, i, format_)
            expected_name = os.path.join(base_dir, image_name)
            actual_name = os.path.join(out_dir, image_name)
    
            err = compare_images(expected_name, actual_name, tol, in_decorator=True)
    
            if not os.path.exists(expected_name):
                raise ImageComparisonFailure("image does not exist: %s" % expected_name)
    
            if err:
                for key in ["actual", "expected"]:
                    err[key] = os.path.relpath(err[key])
>               raise ImageComparisonFailure(
                    "images not close (RMS %(rms).3f):\n\t%(actual)s\n\t%(expected)s " % err
                )
E               matplotlib.testing.exceptions.ImageComparisonFailure: images not close (RMS 0.222):
E               	tests/output_images/Blocks/Surface0.png
E               	tests/baseline_images/Blocks/Surface0.png

tests/tools.py:60: ImageComparisonFailure
```

**Comment by dschwoerer** [comment]
Thanks for the builds and the heads up :+1: 

I will look into it.

**Comment by dschwoerer** [comment]
Resolved by #16
